confluent_platform/README.md

```diff
@@ -0,0 +1,103 @@
+# Agent Check: Confluent Platform
+
+## Overview
+
+This check monitors Confluent Platform and Kafka components through the Datadog Agent.
+
+This integration collects JMX metrics for the following components:
+
+- Broker
+- Connect
+- Replicator
+- Schema Registry
+- SQL Server
+- Streams
+- REST Proxy
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][1] for guidance on applying these instructions.
+
+### Installation
+
+The Confluent Platform check is included in the [Datadog Agent][2] package, so you don't need to install anything else on your Confluent Platform component server.
+
+The check collects metrics via JMX, so you need a JVM on each node so the Agent can run [jmxfetch][3]. It is recommend using an Oracle-provided JVM.
+
+
+### Configuration
+
+1. Edit the `confluent_platform.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your confluent_platform performance data. See the [sample confluent_platform.d/conf.yaml][4] for all available configuration options.
+
+    For each component, you need to create a separate instance to collect its jmx metrics. List of default metrics collected are listed in [`metrics.yaml` file][9]. For example:
+
+    ```yaml
+    instances:
+     - host: localhost
+       port: 8686
+       name: broker_instance
+       user: username
+       password: password
+     - host: localhost
+       port: 8687
+       name: schema_registry_instance
+     - host: localhost
+       port: 8688
+       name: rest_proxy_instance
+    ```
+
+2. [Restart the Agent][5].
+
+#### Containerized
+
+##### Metric collection
+
+For containerized environments, see the [Autodiscovery with JMX][10] guide.
+
+### Validation
+
+[Run the Agent's status subcommand][6] and look for `confluent_platform` under the **JMXFetch** section.
+
+```
+    ========
+    JMXFetch
+    ========
+
+      Initialized checks
+      ==================
+        confluent_platform
+          instance_name : confluent_platform-localhost-31006
+          message :
+          metric_count : 26
+          service_check_count : 0
+          status : OK
+```
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][7] for a list of metrics provided by this check.
+
+### Service Checks
+
+Confluent Platform does not include any service checks.
```

Comment:
Need to update this as `service_checks.json` lists a service check (`confluentplatform.can_connect`).